tensorflow/compiler/tf2xla/kernels/bincount_op.cc

```diff
@@ -0,0 +1,162 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
We should not be adding new files, instead should be editing the file as it existed on the branch, before move

Reply:
This file is created in May 2022, hence reverting from creating a new one.Thank you.